language_service/src/completion.rs

```diff
@@ -765,105 +765,89 @@ fn callable_decl_to_completion_item(
     package_alias_from_manifest: &Option<Arc<str>>,
     callable_namespace: &qsc::hir::Idents,
     imports: &[ImportItem],
-    insert_open_at: Option<Range>,
+    insert_import_at: Option<Range>,
     indent: &String,
-) -> Option<(CompletionItem, SortPriority)> {
+) -> (CompletionItem, SortPriority) {
```

Comment:
I find this function borderline unreadable, but as it was before-hand, will approve to unblock. We should really focus on code clarity and readability more when authoring though, as it makes it really hard to feel confident that you would have spotted any logic bugs when reviewing.

Reply:
I have been working to incrementally improve this, but you're right, it could use a deeper refactor. I filed #1864 to track this.